OptionSelector/CatagoryTableViewController.h

```diff
@@ -0,0 +1,15 @@
+//
+//  CatagoryTableViewController.h
+//  OptionSelector
+//
+//  Created by Jason Wang on 8/14/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface CatagoryTableViewController : UITableViewController
+
+@property (nonatomic) NSArray *categorySections;
```

Comment:
Style: The name of this variable shouldn't indicate the manner in which the data is going to be displayed. I would just call it `categories`.